### DISCUSSION TITLE
通过flink sql连接doris时 delete 和update 不可用?

### DISCUSSION THREAD

**Discussion by lyteyungs** [apache/doris#9443] [A - General / Q&A]
doris官网显示支持flink sql 的  Insert / Update / Delete 事件操作doris,  在执行update 时显示不支持 
![image](https://user-images.githubusercontent.com/7997076/167248566-2b8c3996-279b-4992-b338-ce40b8bb698a.png)
在执行 delete 时不报错 
![image](https://user-images.githubusercontent.com/7997076/167248623-c5aa177d-984c-4e7b-977b-84b616dde51a.png)
但是数据没有删掉?  我已经开启了 批量删除配置 
![image](https://user-images.githubusercontent.com/7997076/167248704-6553cfe2-96de-457d-9a20-8b2f938a7012.png)
在定义表时也指定了'sink.enable-delete' = 'true' 
![image](https://user-images.githubusercontent.com/7997076/167248730-fc483558-e6a1-4e83-a399-afc3a7ce6857.png)
这张表在doris里删除和update都是正常的  ,这是表定义
![image](https://user-images.githubusercontent.com/7997076/167248797-7e4c8fda-43e1-4320-b6b1-a60702f59b4d.png)

**Comment by hf200012** [comment]
update tbl set a=b where id=? This is operated by jdbc, connector does not support this operation
The update of the connector is only on the unique key model, the entire data is updated, and a single update is not supported, and the delete supports the data source of flink cdc

**Comment by JNSimba** [comment]
Flink should not support this syntax
https://nightlies.apache.org/flink/flink-docs-master/zh/docs/dev/table/sql/overview/

<img width="845" alt="image" src="https://user-images.githubusercontent.com/7951521/167250327-c26ce98e-f14c-4802-9efe-17badcc3ba2a.png">
